File: src/main/java/io/github/dqualizer/dqlang/archive/loadtesttranslator/dqlang/modeling/RuntimeQualityAnalysis.java
package io.github.dqualizer.dqlang.archive.loadtesttranslator.dqlang.modeling;

import io.github.dqualizer.dqlang.types.runtimequalityanalysisdefinition.Monitoring;
import lombok.Getter;
import lombok.ToString;

import java.util.LinkedHashSet;

@Getter
@ToString
public class RuntimeQualityAnalysis {

    private LinkedHashSet<Monitoring> monitoring;
    private LinkedHashSet<Object> resilience;
    private LinkedHashSet<ModeledLoadTest> loadtests;
}

File: src/main/kotlin/io/github/dqualizer/dqlang/archive/loadtesttranslator/dqlang/modeling/RuntimeQualityAnalysis.kt
package io.github.dqualizer.dqlang.archive.loadtesttranslator.dqlang.modeling

import lombok.Getter
import lombok.ToString

@Getter
@ToString
class RuntimeQualityAnalysis {
    val monitoring: LinkedHashSet<Any>? = null
    val resilience: LinkedHashSet<Any>? = null
    val loadtests: LinkedHashSet<io.github.dqualizer.dqlang.archive.loadtesttranslator.dqlang.modeling.ModeledLoadTest>? = null
}
